test/conftest.py

```diff
@@ -4,11 +4,12 @@
 import pytest
 import torch
 import torchvision
-from common_utils import CUDA_NOT_AVAILABLE_MSG, IN_FBCODE, IN_OSS_CI, IN_RE_WORKER, OSS_CI_GPU_NO_CUDA_MSG
 
 
 torchvision.disable_beta_transforms_warning()
 
+from common_utils import CUDA_NOT_AVAILABLE_MSG, IN_FBCODE, IN_OSS_CI, IN_RE_WORKER, OSS_CI_GPU_NO_CUDA_MSG
```

Comment:
Did you add this for #7265 or because we actually need it for these tests. I'm not against it, but want to make sure I understand correctly.

Reply:
This is mostly a drive-by, I stole it from https://github.com/pytorch/vision/pull/7282.

I think https://github.com/pytorch/vision/pull/7278 added some v2 imports in `common_utils` and we started seeing the warnings in the test suite, so now we have to disable them before we import anything from `common_utils`.